Fix small typos

elvisangelaccio opened the pull request.

elvisangelaccio committed:
Fix typo

elvisangelaccio committed:
Fix typo

lydiapintscher commented on Frederik.tex:
That is actually fine as far as I can tell.

elvisangelaccio replied:
Not a native speaker here, clearly my english is not good enough :)
I'll drop this change.

elvisangelaccio committed:
Fix another small typo

elvisangelaccio committed:
Fix tense

The sentence is in the paste tense, feels a bit weird to use the present
tense here.

lydiapintscher commented:
Thank you! :)
